Packs/QRadar/Scripts/QRadarCreateAQLQuery/QRadarCreateAQLQuery.py

```diff
@@ -0,0 +1,123 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+from enum import Enum
+from typing import Dict, List
+
+
+REPLACE_KEYS = [
+    ('base_values_to_search', 'base_additional_values'),
+    ('base_fields_to_search', 'base_additional_fields'),
+    ('base_field_state', 'base_additional_field_state'),
+    ('base_field_match', 'base_additional_field_match')
+]
+
+
+class SectionNotFound(Exception):
+    pass
+
+
+class Operators(Enum):
+    OR = 'OR'
+    AND = 'AND'
+
+
+class MatchRule(Enum):
+    EQUAL = "{} = '{}'"
+    NOT_EQUAL = "{} != '{}'"
+    ILIKE = "{} ILIKE '%{}%'"
+    NOT_ILIKE = "{} NOT ILIKE '%{}%'"
+
+
+def fields_section(fields_list: List[str], values_list: List[str], operator: Operators = Operators.OR,
+                   match_rule: MatchRule = MatchRule.EQUAL) -> str:
+    condition_list: List[str] = []
+    for field in map(lambda x: x if ' ' not in x else f"'{x}'", fields_list):
```

Comment:
for simplicity purposes:
```suggestion
    for field in map(lambda x: f"'{x}'", fields_list):
```

Reply:
this solution will cause problems if someone will use a function (e.g. UTF8(payload))